galaxy.yml

```diff
@@ -0,0 +1,35 @@
+namespace: middleware_automation
+name: ocpv-lab
```

Comment:
We probably need to change the collection name to `ocpv_lab` including changing the name of the repository (again)